components/contribution-flow/ContributionBlocker.js

```diff
@@ -85,11 +71,48 @@ export const getContributionBlocker = (loggedInUser, account, tier, shouldHaveTi
     return { reason: CONTRIBUTION_BLOCKER.TIER_EXPIRED, type: 'warning', showOtherWaysToContribute: true };
   } else if (account.settings.disableCustomContributions && !tier) {
     return { reason: CONTRIBUTION_BLOCKER.NO_CUSTOM_CONTRIBUTION, type: 'warning', showOtherWaysToContribute: true };
+  } else if (
+    tier &&
+    tier.interval !== null &&
```

Comment:
Just thinking about it now: `tier.interval` can be `flexible` - which accepts any kind of interval (including null/one-time). There should be no blocker in this case.

Reply:
Makes sense. I've removed those two conditions. 👍🏽

Reply:
@Betree : I spoke too soon. This one actually, do you mean to remove these conditions? In which case the problem is on a one time contribution the flow will be blocked. That's not what we want right? The user having a back account payment method for example should be able to do one-time contributions. 🤔

Reply:
What I meant is that we should not block the flow when `tier.interval === 'flexible'`, because people can still make one-time contributions for them. Basically, the condition should look like:

```es6
cont tierHasFixedInterval = tier => Boolean(tier?.interval && tier.interval !== 'flexible');

(...)

else if (tierHasFixedInterval(tier) && !canMakeRecurringContributions(loggedInUser, account)) {
  return blocker;
}
```

Reply:
@Betree : Ah, thanks for the explanation. 🐼 But this probably wouldn't work, since `tier.interval` has values of nature, `null`, 'month` or `year`. So, what we are doing here is to check for the one time contribution being selected by default. If not, that means it's a recurring contribution and we block the flow. What we have is fixed amounts, and as I understand it we don't have fixed intervals. Does that make sense? 😄 🐨

Reply:
According to https://github.com/opencollective/opencollective-api/blob/deat/platform-tip-debt-transactions/server/graphql/v2/enum/TierInterval.js, `tier.interval` can be:
- `flexible` => flexible (can be either recurring or one-time)
- `month` => fixed recurring
- `year` => fixed recurring
- null => fixed one-time